docs/self-managed/setup/guides/configure-db-custom-headers.md

```diff
@@ -0,0 +1,215 @@
+---
+id: configure-db-custom-headers
+title: "Configure custom headers"
+sidebar_label: "Configure custom headers"
+description: "Learn how to configure DB client custom headers"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+Custom HTTP headers can be added to a component's Elasticsearch and OpenSearch HTTP clients by creating a new Java plugin, and adding the plugin to your Camunda 8 Self-Managed installation. Using custom HTTP headers may be helpful for adding authentication, tracking, or debugging to your database requests.
```

Comment:
Do we need to add for now that it's not supported for Optimize???

Reply:
I was expecting for onboarding Optimize before 8.6. For now, we do not include Optimize config value. @felix-mueller do you think, we should explicitly mention Optimize not included?